client/striplighttimer.py

```diff
@@ -0,0 +1,29 @@
+import requests
+import time
+import datetime
+headers={'accept': 'application/json'}
+payload={'power':'true'}
+mario_API="http://mario.clients.shimley:8000/v1/strip/1/power"
```

Comment:
```suggestion
MARIO_API="http://mario.clients.shimley:8000/v1/strip/1/power"
```
convention is to use UPPER_CASE_NAMES for constant values